src/ContextMenu.tsx

```diff
@@ -69,28 +69,28 @@ export interface IContextMenuView {
 export const ContextMenuView: React.FunctionComponent<IContextMenuView> = props => {
   const { menuItems = [], ...viewProps } = props;
 
+  const { contextMenu: theme } = useTheme();
```

Comment:
:+1: this is actually correct! (fewer hooks could be rendered if we return early).